react-graphql-client-library.md

```diff
@@ -23,31 +23,31 @@ summary = "A tutorial on how to build your own GraphQL client for React applicat
+You may have used a GraphQL client library that was at least partly view layer agnostic, but still able to work with React or other view layer solution in the form of applications like Apollo Client, Urql and Relay. The next chapter is to illustrate how to implement a GraphQL client library that works with React. The takeaway shouldn't be "build your own GraphQL client for your production ready applications", however, you should instead learn:
 
-* see how a GraphQL client library works under the hood
-* see how it isn't too difficult to implement it your way
-* see how you would be able to contribute to the GraphQL ecosystem
+* how a GraphQL client library works under the hood
+* how simple a GraphQL client library is to implement your way
+* how it is capable of contributing to the GraphQL ecosystem
 
-I believe there is great potential to contribute to the GraphQL ecosystem, because it is fairly new. It would be great to have a diverse set of tools out there and not only one ecosystem pushing the whole thing forward. So see it as your opportunity to contribute to it.
+There is contribute to the GraphQL ecosystem, because the tools surrounding it are still fairly new. A diverse set of tools would speed this along, instead of Apollo pushing its maturation forward alone. This is not only a useful addition for your web development skillset, it is also an opportunity to contribute to the early stages of GraphQL.
 
-Before diving into the single parts of implementing your own GraphQL client for React, let's think about the essential parts to be able to consume a GraphQL API in your React application?
+Before diving into implementing your own GraphQL client for React, consider the essentials for consuming a GraphQL API in a React application:
 
-* First, there must be **the GraphQL client itself**. It can be any HTTP library or even the {{% a_blank "native fetch API" "https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API" %}}, but it must be able to send HTTP methods with a payload across the wire. Note: Even though the GraphQL specification isn't opinionated about the transportation layer, the GitHub GraphQL API (which you are going to consume with your own GraphQL client) is using HTTP. Thus our GraphQL client has to be able to execute GraphQL operations by using HTTP methods.
+* A **GraphQL client** must be used. It can be any HTTP library or even the {{% a_blank "native fetch API" "https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API" %}}, but it must be able to send HTTP methods with a payload across the wire.  While the GraphQL specification isn't opinionated about the transportation layer, the GitHub GraphQL API you consume with a GraphQL client is using HTTP. Our GraphQL client must be able to execute GraphQL operations using HTTP methods.
```

Comment:
"Our GraphQL client must be able to execute GraphQL operations using HTTP methods." -> "So because we are using their API, our GraphQL client must be able to execute GraphQL operations using HTTP methods."

Reply:
"Because we are using their API..."  Remember, there is never a need for the word "so" at the beginning of a sentence (nor "thus", "therefore", or "hence").  These are words people say when they're speaking and need to pause, but periods and commas do that in writing.